index.js

```diff
@@ -25,12 +25,39 @@ const panel = require('sdk/panel').Panel({
   }
 });
 
+// Init the panel's location state
+panel.coords = { bottomOffset: -10, leftOffset: 10 };
+
 getActiveView(panel).setAttribute('noautohide', true);
 
-// check for linux here
-// if (system.platform !== 'linux') {
-//   makePanelDraggable(panel);
-// }
+// Draggability seems to work for windows and mac, but not linux.
+if (system.platform === 'winnt' || system.platform === 'darwin') {
+  makePanelDraggable(panel);
+}
+
+function adjustHeight(newHeight) {
+  const xulPanel = getActiveView(panel);
+  const stack = xulPanel.getElementsByTagName('stack')[0];
+  const frame = xulPanel.getElementsByTagName('iframe')[0];
+
+  frame.setAttribute('height', newHeight);
+  stack.setAttribute('height', newHeight);
```

Comment:
This breaks on linux.

I suggest a check above where we call makePanelDraggable.

```js
let isLinux = true;
// Draggability seems to work for windows and mac, but not linux.
if (system.platform === 'winnt' || system.platform === 'darwin') {
  isLinux = false;
  makePanelDraggable(panel);
}
```

then in `adjustHeight`

```js
function adjustHeight(newHeight) {
  const xulPanel = getActiveView(panel);
  const frame = xulPanel.getElementsByTagName('iframe')[0];

  frame.setAttribute('height', newHeight);
  if (!isLinux) {
    xulPanel.getElementsByTagName('stack')[0].setAttribute('height', newHeight);
  }

  xulPanel.sizeTo(320, newHeight);

  redrawPanel();
}
```